content/reviewers_response.md

```diff
@@ -0,0 +1,632 @@
+> 
+> I'm sure you've got this in the github, but just to make the manuscript more readily understandable without digging around in github, do you think you could provide as a supplementary file a list of those 100 most frequently occurring tokens, so that people can get a better feel for what the data is here?
+
+**We have added a supplementary file that provides a listing of 100 tokens for our corpora KL analysis.**
+
+> 10.) https://hypothes.is/a/-2AU8tLmEeu6swvY4jN5Mw
+> 
+> hmmm… not a problem of this manuscript, but that's really not good enough from bioRxiv is it? Change one word of a long and complex title and suddenly 'oh, we can't do it'. A comment to suggest that bioRxiv could do better would be fun, no(?) i.e. look at author names AND title and if both are similar enough, then do auto-linking.
+> 
+> oh okay, you did actually do that. Nice :)
+
+**We are glad that you liked our document matching solution.**
+
+> 11.) https://hypothes.is/a/dhGbgNLnEeuYkkfUNqKwgA
+> 
+> I don't suppose you could possibly be precise about this rather than just 'a limited number'? Is it 5, 50, or 500?
+
+**We agree that this line is vague and have updated our text to be more explicit.**
+
+```diff
+- There were a limited number of cases in which authors appeared to post preprints after the publication date, which results in preprints receiving a negative time difference, as previously reported [...]
+
++ We encountered 123 cases where the preprint posting date was subsequent to the publication date, resulting in a negative time difference, as previously reported [...].
+```
+
+> 12.) https://hypothes.is/a/DzrzGtLtEeuJlcdOlUfUBA
+> 
+> I'm surprised to see no citation given to JANE: https://jane.biosemantics.org/
+>
+>https://academic.oup.com/bioinformatics/article/24/5/727/202224
+>
+>reviewed in https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6300233/
+>
+>The 'find journals' functionality of JANE appears somewhat similar to the discovering similar journals functionality here.
+
+**Thank you for pointing this citation out. 
+We have incorporated it into our manuscript.**
+
+```diff
+- We developed a web application that places any bioRxiv or medRxiv preprint into the overall document landscape and identifies similar papers and journals.
+
++ We developed a web application that places any bioRxiv or medRxiv preprint into the overall document landscape and identifies topically similar papers and journals (similar to [..]).
+```
+
+> 13.) https://hypothes.is/a/eJ7-DtLpEeuh0A8OL8WAWw
+> 
+> This is the kind of journal-faff difference that I hypothesise would not be visible or less visible if one did an analysis of preprints vs author manuscripts.
+>
+>There is change but is change from figure to 'fig.' to suit journal style actually helpful/valuable? In my opinion it is not!
+
+**We recognize the limitation of our design choice and have mentioned using accepted author manuscripts as an alternative for future analyses.**
```

Comment:
```suggestion
**We recognize the limitation of our design choice and agree it would be interesting to compare journal-supplied vs author-supplied manuscripts in a future study.**
```